src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -88,20 +88,20 @@ class TimeMeasurement
     clock::time_point start_;
     clock::time_point end_;
 };
-bool OPT_AppelDuSimplexe(const OptimizationOptions& options,
-                         PROBLEME_HEBDO* problemeHebdo,
-                         int NumIntervalle,
-                         const int optimizationNumber,
-                         const OptPeriodStringGenerator& optPeriodStringGenerator,
-                         IResultWriter& writer)
+
+static bool OPT_TryToCallSimplex(const OptimizationOptions& options,
```

Comment:
Instead of passing `solveTime` by non-constant reference, it may be clearer to return a `struct { bool ok, long long solveTime}`.